docs/developers/contributing/performance/profiling.md

```diff
@@ -2,12 +2,22 @@
 
 # Profiling
 
-In contrast to performance tracing, profiling does not provide information
-about the timing of events, but instead provides combined information about the
-total time of execution of each function. It also produces a call graph that
-simplifies understanding the call relationship between functions.
-
-![A section of an example call graph showing the functions called in napari's layerlist and some of their profile statistics such as call count and cumulative percentage time spent.](images/execution_graph.png)
+Profiling provides information about the
+total time of execution of each function when performing a specific task or workflow.
+It produces a complete call graph that
+simplifies understanding the call relationship between functions. It is useful for
+identifying which functions are taking most of the runtime and causing bottlenecks.
+
+[Performance monitoring](perfmon), unlike profiling provides information about the
+timing of events or specific functions each time it is called. It can be useful when
+profiling identifies a common function to be slow, but
+as this function gets called from many places so it's hard to know where to optimize.
+Performance monitoring is able to tell you at which points during execution it is
+called and how long it took each time. This is also helpful in differentiating
+functions that are only slow in the first call versus functions that are consistently
+slow. See [performance monitoring](perfmon) for more details.
+
+![A section of an example call graph showing the functions called in napari's layerlist and some of their profile statistics such as call count and cumulative percentage time spent.](../images/execution_graph.png)
```

Comment:
```suggestion
![A section of an example call graph showing the functions called in napari's layerlist and some of their profile statistics such as call count and cumulative percentage time spent.](../../images/execution_graph.png)
```
The images are in /developers/images
and this file is in:
/developers/contributing/performance
So I think I have it correct now?

Reply:
It's that or we move the images folder.